README.md

```diff
@@ -55,128 +58,75 @@ package:
+In case you only want to work with NetworKit's C++ core, this section describes how to build it.
+We recommend [CMake] and your preferred build system for building the C++ part of NetworKit.
 
 The following description shows how to use [CMake] in order to build the C++ Core only:
 
-First you have to create and change to a build directory: (in this case named `build_lib`)
+First you have to create and change to a build directory: (in this case named `build`)
 
-	mkdir build_lib
-	cd build_lib
+	mkdir build
+	cd build
 
 Then call [CMake] to generate files for the `make` build system, specifying the directory of the root `CMakeLists.txt` file (e.g., `..`). After this `make` is called to start the build process:
 
 	cmake ..
 	make -jX
 
-To speed up the compilation with make a multicore machine, you can append `-jX` where X denotes the number of threads to compile with.
+To speed up the compilation with make a multi-core machine, you can append `-jX` where X denotes the number of threads to compile with.
 
-[CMake] offers a variety of options to customize the build to your needs (read [CMake] documentation for more options).
+### Use NetworKit as a library
 
-**NOTE FOR OSX**: [CMake] aims at automatically finding a suitable C++ Compiler for building networkit.
-However, the default compiler on MacOS is a Apple LLVM clang like compiler which has no built-in OpenMP support and is linked against all common compiler calls. In order to compile networkit properly, you have to to specify your desired compiler and the version (we recommend a GNU-based compiler) such as:
+To use the previous compiled networkit library, you need to have it installed, and link
+it while compiling your project.
+To install NetworKit into a specific directory, you can specify with the cmake
+`INSTALL_PREFIX` flag:
 
-	cmake -DCMAKE_CXX_COMPILER=g++-8 ..
+    cmake -DCMAKE_INSTALL_PREFIX=<path/to/install/directory> ..
+    make -jX install
 
-### Use NetworKit as a library
+Once NetworKit has been installed, you can use include directives in your
+C++\-application as follows:
 
-To use the previous compiled networkit library, you have to link it while compiling your project. The include directives in your C++\-application should look like the following
+	#include <networkit/graph/Graph.hpp>
 
-	#include <NetworKit/graph/Graph.h>
+You can compile your source as follows:
 
-By default, [CMake] creates a symlink to the source tree inside your build directory. `NetworKit` in the directory `include `is a symlink to the directory `networkit/cpp`, so the directory structure from the repository is valid. To compile your application, you need to add the paths for the header files and the location of the library. There is a simple source file to demonstrate this. Feel free to compile `LibDemo.cpp` as follows:
+	g++ my_file.cpp -std=c++11 -I/path/to/install/directory/include -L/path/to/install/directory -lnetworkit
```

Comment:
This will only work if the library path is in `LD_LIBRARY_PATH`. Since this is probably complex to explain (and also unnecessary), should we just demonstrate how a global installation (in `/usr/local`) works?